app/models/pet.rb

```diff
@@ -30,9 +30,16 @@ def self.all_pets_by_adoption_status
 
   def self.pets_with_approved_apps
    Pet.joins(:pet_applications)
+      .distinct
       .where('pet_applications.approved = ?', 'true')
   end
 
+  def self.pets_with_unapproved_apps
+   Pet.joins(:pet_applications)
+      .distinct
```

Comment:
Good call, I had originally thought I was seeing repetition because we had just added a bunch of pets with the same name, I see now that this was why.